### DISCUSSION TITLE
Dynamic Page Cache Problem

### DISCUSSION THREAD

**Discussion by tlgsn** [vercel/next.js#47484] [Help]
### Summary

Hi,

I have folders named /[product] and /panel within the /app folder. In the pages within [product], I fetch and display product information from the database, and when I make a change in the database, it is immediately reflected when I switch between pages as expected.

The /panel folder is the user panel route. When users log in, they go here first. However, I have a problem with this file path. For example, if a user updates their information, when they come back to this page, cached data is still displayed. The data is only updated if they refresh the page with F5. So the page does not update when it comes via <Link>.

For example,
const timestamp = new Date().getTime();

and make the page content as follows:
<>
{timestamp}
</>

the timestamp doesn't update even if I update the page content. I have to refresh with F5.

Have you encountered this problem before? Or do you know of a solution?

### Additional information

```js
next: ^13.2.4
node: v16.18.0
```

**Comment by icyJoseph** [comment]
Could you try adding `export const dynamic = "force-dynamic"` to that page?

**Reply by tlgsn** [reply]
I added it but unfortunately it doesn't work.

https://codesandbox.io/p/sandbox/cevztd?file=%2Fapp%2Fpanel%2Fpage.jsx
